_rules/SC2-1-2-no-keyboard-trap.md

```diff
@@ -0,0 +1,241 @@
+---
+name: No keyboard trap
+rule_type: composite
+description: |
+  This rule checks if it is possible to use standard keyboard navigation to navigate through all content on a web page without becoming trapped in any element, or that it is possible to use non-standard keyboard navigation to navigate through content where focus is trapped when using standard ways of keyboard navigation.
+
+success_criterion:
+- 2.1.2 # No Keyboard Trap (A)
+
+atomic_rules:
+- [No keyboard trap standard navigation](SC2-1-2-no-keyboard-trap-standard-navigation)
+- [No keyboard trap non-standard navigation](SC2-1-2-no-keyboard-trap-non-standard-navigation)
+
+authors:
+- Geir Sindre Fossøy
+- Dagfinn Rømen
+- Anne Thyme Nørregaard
+---
+
+## Aggregation Definition
+
+### Applicability
+
+The rule applies to any HTML or SVG element on a web page that is [focusable](#focusable).
```

Comment:
Do we need "on a web page" in the sentence? I think several other rules leave that out.

Reply:
This is based on the the two atomic rules. I suggest we keep it in the composite rule as well.

Reply:
I suggest removing it from the other two rules. It is an easy fix editorially but would avoid problems and limitations down the line.